src/plugins/vis_default_editor/public/components/sidebar/controls.tsx

```diff
@@ -131,19 +134,32 @@ function DefaultEditorControls({
             defaultMessage: 'Auto updates the visualization on every change.',
           })}
         >
-          <EuiButtonToggle
-            label={i18n.translate('visDefaultEditor.sidebar.autoApplyChangesAriaLabel', {
-              defaultMessage: 'Auto apply editor changes',
-            })}
+          <EuiButton
             className="visEditorSidebar__autoApplyButton"
             data-test-subj="visualizeEditorAutoButton"
-            fill={autoApplyEnabled}
             iconType="refresh"
-            isSelected={autoApplyEnabled}
-            onChange={toggleAutoApply}
+            color={autoApplyEnabled ? 'primary' : 'text'}
+            fill
+            onClick={() => toggleAutoApply(!autoApplyEnabled)}
             size="s"
-            isIconOnly
-          />
+            minWidth={80}
+          >
+            <EuiScreenReaderOnly>
+              <span>
+                {i18n.translate('visDefaultEditor.sidebar.autoApplyChangesLabel', {
+                  defaultMessage: 'Auto apply is',
```

Comment:
Not 100% sure about this setup - should "Auto apply is on" and "Auto apply is off" be separate phrases so they can theoretically change order in a translation?

Reply:
I checked using google translate (I know, I know) and it seems like that's the case for Korean:
<img width="663" alt="Screenshot 2020-10-30 at 16 54 44" src="https://user-images.githubusercontent.com/1508364/97727475-bc52e100-1ad0-11eb-84c1-31e8b2f3bf95.png">
<img width="657" alt="Screenshot 2020-10-30 at 16 55 28" src="https://user-images.githubusercontent.com/1508364/97727479-be1ca480-1ad0-11eb-8a50-22c292504495.png">

Reply:
cc @myasonik

Reply:
cc @cchaos

Reply:
Fair point, I'll try to fix

Reply:
Fixed: https://github.com/elastic/kibana/pull/81499/commits/2d0d7dc383fc5c68f9879e1a7e2bb9dc350fc3d8

Reply:
Nice solution 👍

Reply:
Good catch @flash1293! Given that we don't translate into very many languages, I think google translate is a fair test for users.

@cchaos - fix looks great 👍